packages/scripts/scripts/lint-pkg-json.js

```diff
@@ -38,7 +41,7 @@ const defaultIgnoreArgs = ! hasIgnoredFiles ?
 
 const result = spawn(
 	resolveBin( 'npm-package-json-lint', { executable: 'npmPkgJsonLint' } ),
-	[ ...config, ...defaultIgnoreArgs, ...args ],
+	[ ...defaultConfigArgs, ...defaultIgnoreArgs, ...args, defaultFilesArgs ],
```

Comment:
Guessing this is meant to be spread `...defaultFileArgs`

Reply:
🙈